pyppeteer/page.py

```diff
@@ -753,8 +752,14 @@ def _onBindingCalled(self, event: Dict) -> None:
         except Exception as e:
             helper.debugError(logger, e)
 
-    def _addConsoleMessage(self, type: str, args: List[JSHandle]) -> None:
-        if not self.listeners(Page.Events.Console):
+    def _addConsoleMessage(
+            self,
+            type: str,
+            args: List[JSHandle],
+    ) -> None:
+        # TODO puppetter also takes stacktrace argument but it seems that
+        # in python it's not necessary?
```

Comment:
Not exactly sure what the intent was in puppeteer, but I think it was to provide a stacktrace for executed js within the browser js, so I think it could still be useful, but not sure at this point.